AdminWebsite/AdminWebsite/ClientApp/src/app/suitability/suitability.module.ts

```diff
@@ -5,12 +5,17 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule, Routes } from '@angular/router';
 import { SharedModule } from '../shared/shared.module';
-import { VhOfficerAdminGuard } from '../security/vh-officer-admin.guard';
 import { AnswerListEntryComponent } from './answer-list-entry/answer-list-entry.component';
 import { ScrollableSuitabilityAnswersService } from './services/scrollable-suitability-answers.service';
 import { QuestionnaireMapperFactory } from './services/questionnaire-mapper-factory.service';
 
-export const routes: Routes = [{ path: 'questionnaire', component: AnswersListComponent, canActivate: [VhOfficerAdminGuard] }];
+export const routes: Routes = [
+    {
+        path: 'questionnaire',
+        redirectTo: 'dashboard' // component: AnswersListComponent,
```

Comment:
Are these comments needed?

Just a minor one - have approved anyway :)

Reply:
Hey - I thought if we decided we wanted to bring back the questionnaire results page in the future, we just need to remove the redirect and uncomment this code. I thought it might be helpful, but also I know leaving commented out code usually considered a 'bad practice'. So happy to remove it altogether if you prefer.

Reply:
No that sounds good to me, feel free to leave it in :)